cpu/nrf52/Makefile.features

```diff
@@ -12,6 +12,13 @@ ifneq (,$(filter nrf52811xxaa nrf52820xxaa nrf52833xxaa nrf52840xxaa,$(CPU_MODEL
   FEATURES_PROVIDED += netif_openwsn
 endif
 
+# The USBD peripheral is not available on all SoCs
+ifneq (,$(filter nrf52820xxaa nrf52833xxaa nrf52840xxaa,$(CPU_MODEL)))
+  FEATURES_PROVIDED += periph_usbdev
+  # the nrf usb stack is supported by tinyUSB
+  FEATURES_PROVIDED += tinyusb_device
+endif
+
```

Comment:
I think we have an issue here.
`periph_usbdev` is designed to be used by `USBUS` only.
So you must not provide this feature alongside with `tinyusb_device`.
`tinyUSB` provides its own low level peripheral driver.

Reply:
perhaps we can  try to summon @gschorcht for its guidance here.

Reply:
Ah really, I understood it as "the CPU has a usb peripheral", which is also what is documented here: https://doc.riot-os.org/md_doc_2doxygen_2src_2feature__list.html#autotoc_md2318

Reply:
Anyway the boards before also exposed `periph_usbdev` and using tinyusb was perfectly possible.

Reply:
In fact, what @gschorcht wanted to do was:
use USBUS by default for `highlevel_stdio`
but, if for whatever reason, `tinyUSB` is used by user switch the USB stack to tinyUSB and thus use `stdio_tinyusb_cdc_acm`.
In the end, the board can provide `periph_usbdev` but if `tinyUSB` is used, `USBUS` must be discarded.

Reply:
to sum up, I think it's safe to provide `tinyusb_device` for nrf52 based board (that have USB) but let USBUS as default stack. Users can then chose to use tinyUSB by simply using FEATURES_REQUIRED+="tinyusb_device".
If `highlevel_stdio` is also there,  `stdio_tinyusb_cdc_acm` should be pick automatically.

Reply:
But doesn't `tinyusb_device` only say that the tinyUSB stack is supported and can be used on user wish? https://doc.riot-os.org/md_doc_2doxygen_2src_2feature__list.html#autotoc_md2329

So I'd argue the CPU can provide both, `usbus` is still used as default, but the user could decide to enable tinyUSB with `USEPKG += tinyusb` (without providing the feature, the build system disallows this).

Reply:
Moreover, is it really a good idea to define the USB at CPU level ?
I think it's done at board level currently because a board doesn't always expose USB pins.

Reply:
You can confirm this with `make BOARD=feather-nrf52840-sense -C examples/hello-world info-modules` which still includes `usbus`.

Reply:
> Moreover, is it really a good idea to define the USB at CPU level ? I think it's done at board level currently because a board doesn't always expose USB pins.

That is indeed a good point.

So maybe we rather want the following: each board defines `periph_usb` if present, and the cpu can check for `periph_usbdev` and then additionally set `tinyusb_device`?

Reply:
> So maybe we rather want the following: each board defines periph_usb if present, and the cpu can check for periph_usbdev and then additionally set tinyusb_device?

Let's try this way.

Reply:
See the new commit (sorry for the force-push).